providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/ResolveStrategy.java

```diff
@@ -0,0 +1,13 @@
+package dev.openfeature.contrib.providers.flagd;
+
+import com.google.protobuf.Message;
+
+import java.util.function.Function;
+
+/**
+ * Request to Response resolving strategy.
+ * */
+public interface ResolveStrategy {
```

Comment:
```suggestion
interface ResolveStrategy {
```

These should all be package private, I think.

Reply:
Good one, yes . I will update with next commit :)

Reply:
corrected with the latest commit :)